logos/docker-compose.yaml

```
@@ -0,0 +1,109 @@
+services:
+  logos-server:
+    restart: unless-stopped
+    container_name: logos-server
+    build:
+      context: ../
+      dockerfile: ./logos/Dockerfile
+    depends_on:
+      logos-db:
+        condition: service_healthy
+    volumes:
+      -   data_volume:/src/logos
```

YAMLlint (1.37.1)

[warning] too many spaces after hyphen

(hyphens)